src/client.ts

```diff
@@ -72,11 +73,20 @@ class Client extends EventEmitter implements IClient {
 
         this.metrics.on('error', (msg) => this.logger.error(`metrics: ${msg}`));
         this.unleash.on('error', (msg) => this.logger.error(msg));
+
         this.unleash.on('ready', () => {
             this.emit('ready');
             this.ready = true;
             this.metrics.start();
         });
+        this.unleash.on(UnleashEvents.Unchanged, () => {
+            lastMetricsFetch.set(new Date().getTime());
+        });
+        this.unleash.on(UnleashEvents.Changed, () => {
+            const updated = new Date().getTime();
+            lastMetricsFetch.set(updated);
+            lastMetricsUpdate.set(updated);
+        });
     }
```

Comment:
Other important behavior change, adds an event listener to Unchanged updating the fetch counter, and to Changed events updating both last fetch and last update.